doc/ja/lineedit.txt

```diff
@@ -1022,7 +1022,7 @@ ifndef::basebackend-html[`@`]
 --
 
 vi-edit-and-accept::
-エディタとして vi を起動し、バッファの内容を編集させます。エディタが終了すると編集後の内容をバッファに反映した後 accept-line コマンドと同じ動作をします。ただしエディタの終了ステータスが 0 でないときは何も行いません。
+エディタとして VISUAL、EDITOR、または vi を起動し、バッファの内容を編集させます。エディタが終了すると編集後の内容をバッファに反映した後 accept-line コマンドと同じ動作をします。ただしエディタの終了ステータスが 0 でないときは何も行いません。
```

Comment:
I know a small amount of Japanese. Please let me know if the translation is incorrect and needs changing!

Reply:
Perfect translation!